### PR TITLE
Error display component

### PR DESCRIPTION
**Pull request by 1emu**
Component shows on error

image

image

Expands showing received error message

image

image

image

### CODE REVIEW

File: src/components/Error/ErrorMessage.css [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  vertical-align: middle;
+  padding: 0 6px;
+  font-weight: 400;
+  font-size: 11px;
+  line-height: 26px;
+  letter-spacing: -0.2px;
+  text-transform: uppercase;
+  color: #736e7d;
+  background: #fff;
+  border: 1px solid rgba(115, 110, 125, 0.5);
```
Comment (ncomerci) [inline src/components/Error/ErrorMessage.css:73]: This color is already defined as `--dark-gray-rgb`, you can write it as `rgba(var(--dark-gray-rgb), 0.5)` until we define with juani a standard for the opacity

File: src/intl/en.json [hunk 2]
```diff
@@ -43,6 +43,13 @@
     "button": "Go Back"
   },
   "error": {
+    "message": {
+      "hide": "Hide error",
+      "show": "Show error",
+      "copy": "Copy to Clipboard",
+      "copied": "Copied!",
+      "call_for_action": "We rely on an external service that *can* fail. However, feel free to post this error message on [Discord](https://discord.com/channels/894658869391933540/987439629533855774)."
```
Comment (ncomerci) [inline src/intl/en.json:51]: I would "We rely on external services that can fail..." (in plural)

**Review by ncomerci — APPROVED** [review]
🥇

File: src/components/Error/ErrorMessage.css (hunk 1 above)
Comment (1emu) [reply src/components/Error/ErrorMessage.css:73]: It's not defined in master :/

Maybe when you rebase you can replace the occurrences? 🙏

**Pull request merged**